src/computeHashSum.in.lua

```diff
@@ -169,7 +169,7 @@ function main()
    end
    fh:write(s)
    if (HashSum:sub(1,1) == "@" ) then
-      HashSum = find_exec_path("sha1sum")
+      HashSum = find_exec_path("sha1sum") or find_exec_path("shasum")
```

Comment:
Another way to say this is that the test suite run the tests without installing Lmod.  Instead it runs them directly from the source.